eZ/Publish/API/Repository/Tests/UserServiceAuthorizationTest.php

```diff
@@ -297,6 +297,70 @@ public function testUpdateUserThrowsUnauthorizedException()
         /* END: Use Case */
     }
 
+    /**
+     * Test for the updateUserPassword() method.
+     *
+     * @see \eZ\Publish\API\Repository\UserService::updateUserPassword()
+     */
+    public function testUpdateUserPasswordThrowsUnauthorizedException()
+    {
+        $this->expectException(\eZ\Publish\API\Repository\Exceptions\UnauthorizedException::class);
```

Comment:
Since I'm already working on this file I could do the update. Most of the comments regarding introduced tests result from me trying to keep to style consistent with other in the same file.

@alongosz :+1: / :-1: ?

Reply:
> Since I'm already working on this file I could do the update. Most of the comments regarding introduced tests result from me trying to keep to style consistent with other in the same file.

Depends on the diff. If it's too big -> separate PR. However new code should be aligned already, hence my comment

Reply:
A separate PR it is then.